plugins/destination/mysql/client/schema.go

```diff
@@ -0,0 +1,145 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func identifier(name string) string {
+	return fmt.Sprintf("`%s`", name)
```

Comment:
```suggestion
	return "`" + name + "`"
```
for the sake of speed

Reply:
Unless you have benchmarks that can show a significant improvement, I'd rather keep the current code for readability